bin/ruby-build

```diff
@@ -1219,6 +1217,10 @@ if [ -n "$RUBY_BUILD_SKIP_MIRROR" ] || ! has_checksum_support compute_sha2; then
   unset RUBY_BUILD_MIRROR_URL
 fi
 
+ARIA2_OPTS="${RUBY_BUILD_ARIA2_OPTS} ${IPV4+--disable-ipv6=true} ${IPV6+--disable-ipv6=false}"
+CURL_OPTS="${RUBY_BUILD_CURL_OPTS} ${IPV4+--ipv4} ${IPV6+--ipv6}"
+WGET_OPTS="${RUBY_BUILD_WGET_OPTS} ${IPV4+--inet4-only} ${IPV6+--inet6-only}"
```

Comment:
At least for aria2, some user may want to specify custom options for splitting & parallelism for better download performance. Because it'd be depending on environment, I'd like to allow user to specify custom configurations via environment.

For ipv4/ipv6 options, I just moved them and I have never used it actually....